content/en/apps/guides/data/on-boarding-data-integrity.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Best Practices for Data Integrity During On-Boarding"
+linkTitle: "Data Integrity in On-Boarding"
+weight: 15
+description: >
+    How to ensure training data ends up in training and production data ends up production.
+
+---
+
+{{% pageinfo %}}
+This best practices guide should not be followed verbatim.  Instead, it is important to review each suggestion and assess if your deployment can benefit from it.
+{{% /pageinfo %}}
```

Comment:
This block may be better as a note, but we can revisit after the main structure/overview is addressed.